src/styles/components/hero.sass

```diff
@@ -0,0 +1,24 @@
+@import '../common'
+
+$font-color: #fff
```

Comment:
We defined a bunch of color and font-size variables on `_variables.sass`, you'd better to reuse them instead of re-defined here again.